webvtt_internal_test.go

```diff
@@ -35,3 +37,42 @@ func TestParseTextWebVTT(t *testing.T) {
 		assert.Equal(t, "Incorrect end tag", s.Items[0].Text)
 	})
 }
+
+func TestTimestampMap(t *testing.T) {
+	for i, c := range []struct {
+		line           string
+		expectedOffset time.Duration
+		expectError    bool
+	}{
+		{
```

Comment:
Could you add cases where :

- `local` is not empty and `mpegts` is empty
- both `local` and `mpegts` are not empty